toc/rfc/README.md

```diff
@@ -0,0 +1,32 @@
+# Cloud Foundry Request For Comments
+
+Cloud Foundry community members use this directory as a forum to make, discuss and record both process and technical decisions. The outcomes of these discussions can be either an Action Plan, a new Standard or Process that the Cloud Foundry community should follow. This repository is open as a reference for all working groups and for the wider community.
+
+## Process
+
+1. Create a new branch on this repo and copy `rfc-000-template.md` to `rfc-000-my-proposal.md` and edit. The numbering used should be the next sequential number that is available when you create your PR.
+1. If your RFC is only relevant for a single Working Group, please place it in a directory for the working group within toc/rfc.
+   1. If your RFC is only relevant for a sigle Working Group, and the Working Group hosts RFCs elsewhere and this step has not already been completed, please create a directory within toc/rfc and place a link to where the RFCs can be found to aid in discoverability and follow the already agreed process. 
```

Comment:
```suggestion
   1. If that Working Group hosts its RFCs elsewhere, its RFC subdirectory will contain a file called `EXTERNAL.md` that contains a link to that group's RFC repository. In that case, follow that link to create the group-specific RFC in that repository.
```
This external-redirect file seems like something that the TOC can backfill for existing WGs (which I expect is only Paketo) and can create contemporaneously with any new or altered WGs.